tests/upgrades-images-unsigned/upgrade_test.go

```diff
@@ -15,12 +15,9 @@ var _ = Describe("cOS Upgrade tests - Images unsigned", func() {
 		s.EventuallyConnects(360)
 	})
 
-	AfterEach(func() {
```

Comment:
This suite is always resetting to verify that the version did actually changed, so I've just removed this.. should go a bit faster